### PR TITLE
Feat/initial db server

### PR DESCRIPTION
**Pull request by AKhoo**
Completed DB seeding script and server API endpoints.

### CODE REVIEW

File: db/config.js [hunk 1]
```diff
@@ -11,291 +15,5 @@ db.once('open', () => {
 
 var Schema = mongoose.Schema;
```
Comment (ccades) [inline db/config.js:17]: Good call on the separation... pro move!

File: server/index.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
 const express = require('express');
 const app = express();
 const port = 3456;
-var stocks = require('../db/config');
+var db = require('../db/config');
+var stocks = require('../db/stock');
+
+
+// ============================================
```
Comment (ccades) [inline server/index.js:8]: Love the major separations... makes things very clear

**Review by ccades — APPROVED** [review]
lgtm

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+const express = require('express');
+const app = express();
+const port = 3456;
+var db = require('../db/config');
```
Comment (therobinkim) [inline server/index.js:4]: don't seem like `var db` is being used anywhere. also, maybe more consistency w/ `var` vs `const`

File: db/stock.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+Stock.listAll = () => {
+  // Returns a Promise object which resolves to a list of all stocks in the collection.
+  return Stock.find({}, 'ticker name');
+};
+
+Stock.getInfo = ticker => {
+  // Returns a Promise object which resolves to basic info for the stock matching the provided ticker.
+  return Stock.findOne(
+    {ticker: ticker}, 
+    'ticker name analystBuy platformOwners');
```
Comment (therobinkim) [inline db/stock.js:57]: indentation seems wonky

suggestion: 
```
return Stock.findOne(
  {ticker: ticker}, 
  'ticker name analystBuy platformOwners'
);
```

(and same for the others below)

**Pull request merged**